lib/ansible/modules/packaging/os/apt.py

```diff
@@ -157,11 +157,9 @@
   apt:
     name: foo
 
-- name: Install a list of packages
+- name: Install multiple packages
```

Comment:
instead of changing the example, i would add an additional one, both ways are valid and there are reasons to use a var instead of putting the list directly. Also 'a list' is used on purpose to make clear the 'type' of the option is `list`